.gitignore

```diff
@@ -11,3 +11,5 @@ __pycache__
 CMakeFiles
 cmake_install.cmake
 CMakeCache.txt
+data/terrain/.DS_Store
+.DS_Store
```

Comment:
You can just add `**/.DS_Store` I think which should capture all of those files.